.clang-tidy

```diff
@@ -25,25 +31,31 @@ Checks: >
   -hicpp-special-member-functions,
   -hicpp-use-equals-default,
   -hicpp-vararg,
-  -hicpp-vararg,
   -llvm-header-guard,
   -llvm-include-order,
+  -llvm-qualified-auto,
```

Comment:
I think this one is actually useful. I've seen many bugs in the past because these were not clear. Perhaps we should start enforcing this at least for new code?

Reply:
https://github.com/facebookincubator/velox/pull/15135/files#r2433480584